src/Actions/MenuEditor.php

```diff
@@ -198,7 +198,10 @@ public function remove_unused_menu_manager_post_type() {
      */
     public function add_editor_menu_capabilities() {
 
-        if ( current_user_can( 'editor' ) ) {
+        $current_user = wp_get_current_user();
+
+        if ( current_user_can( 'editor' ) && ( ! $current_user->caps['administrator'] ) ) {
```

Comment:
I'm assuming that an admin also meets the first criteria, but you don't want to add this cap if they are admins.

Reply:
Correct